e2e/pages/events/manageOrders.page.js

```diff
@@ -78,6 +78,11 @@ const selectChildren = async (option, indexes = []) => {
   await I.runAccessibilityTest();
 };
 
+const enterTitle = async (text) => {
+  I.fillField(section4.title, text);
+  await I.runAccessibilityTest();
```

Comment:
Don't need accessibility on this, only need it on the directions. The accessibility tests run against the whole page